index.html

```diff
@@ -203,105 +203,123 @@ <h2>Multi-Post <br>
-                                    <img src="img/rightArrow.png" alt="right arrow">
+                            </li>
+                            <li class="list-flex ">
+                                <a>
+                                    <img src="img/html.png" alt="Html">
+                                </a>
+                                <div class="name-prom">
+                                    Html
+                                </div>
+                            </li>
+                            <li class="list-flex ">
+                                <a>
+                                    <img src="img/css.png" alt="CSS">
+                                </a>
+                                <div class="name-prom">
+                                    Css
                                 </div>
+                            </li>
+                        </ul>
+                        <div class="language-title">
+                            <div class="title1">Frameworks</div>
+                            <div class="arrow">
+                                <img src="img/rightArrow.png" alt="right arrow">
                             </div>
-                            <div class="separator1">
-                                <img src="img/separator.png" alt="separator">
+                        </div>
+                        <div class="separator">
+                            <img src="img/separator.png" alt="separator">
+                        </div>
+                        <div class="language-title">
+                            <div class="title1">Skills</div>
+                            <div class="arrow1">
+                                <img src="img/rightArrow.png" alt="right arrow">
                             </div>
                         </div>
-                    </li>
-                </ul>
-            </div>
+                        <div class="separator1">
+                            <img src="img/separator.png" alt="separator">
+                        </div>
+                    </div>
+                </li>
+            </ul>
         </div>
+        <section class="contact">
+            <article class="contact_footer">
+                <h2>Contact me</h2>
+                <p>If you have an application you are interested
+                    in developing, a feature that you need built or a project
+                    that needs coding. <br>I'd
+                    love to help with it!</p>
+                <form class="form-start" action="https://formspree.io/f/xlekpydn" method="POST">
+                    <input type="name" type="text" name="name" class="name" placeholder="Lorem Ipsum" maxlength="30"
+                        required>
+                    <input type="email" type="email" name="email" class="email" placeholder="loremipsum@gmail.com"
+                        required>
+                    <textarea type="message" type="text" name="message" class="message"
```

Comment:
- [ ] The textarea tag doesn't require a type, only `input` tags require a type. please kindly make the necessary change by remove the type attribute. :+1: